From be0e853408a6278fa62f43864d90bf2f53170734 Mon Sep 17 00:00:00 2001
From: Yuta Saito <kateinoigakukun@gmail.com>
Date: Sat, 10 Aug 2024 12:26:08 +0000
Subject: [PATCH] Remove the workaround for WASI errno conflict

As we have fixed the conflict in the WASILibc overlay module while
porting swift-foundation, we can remove the workaround from
corelibs-foundation too.
---
 Sources/Foundation/NSPathUtilities.swift | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/Sources/Foundation/NSPathUtilities.swift b/Sources/Foundation/NSPathUtilities.swift
index f6cda6abf9..408b18bebf 100644
--- a/Sources/Foundation/NSPathUtilities.swift
+++ b/Sources/Foundation/NSPathUtilities.swift
@@ -12,13 +12,6 @@
 import WinSDK
 #elseif os(WASI)
 import WASILibc
-// CoreFoundation brings <errno.h> but it conflicts with WASILibc.errno
-// definition, so we need to explicitly select the one from WASILibc.
-// This is defined as "internal" since this workaround also used in other files.
-internal var errno: Int32 {
-    get { WASILibc.errno }
-    set { WASILibc.errno = newValue }
-}
 #endif
 
 #if os(Windows)
